kombu/transport/redis.py

```diff
@@ -1429,7 +1429,7 @@ def _sentinel_managed_pool(self, asynchronous=False):
 
         return sentinel_inst.master_for(
             master_name,
-            self.Client,
+            redis.StrictRedis,
```

Comment:
`StrictRedis` [is deprecated](https://github.com/redis/redis-py/blob/b5d4d29409f83e9a0056c6ed12414bf58e1ea1c0/CHANGES#L583).
Better use Redis instead.